src/bootstrap.ts

```diff
@@ -0,0 +1,45 @@
+import PeerInfo from "peer-info";
+import PeerId from "peer-id";
+import Multiaddr from "multiaddr";
+import dns from 'dns'
+
+type ServerMap = Map<string, PeerInfo>
+
+/** Load Bootstrap node addresses.
+*   - If a string of comma separated multiaddrs is passed, use this first
+*   - If there are ENV Variables, use them second
+*   - Otherwise look at the DNS entry for our testnet.
+*/
+export async function getBootstrapAddresses(addrs?: string): Promise<ServerMap> {
+  let addresses: string[]
+  let bootstrapServerMap = new Map<string, PeerInfo>()
+  let addr: Multiaddr
+
+  if (addrs){
+    addresses = addrs.split(',')
+  } else if (process.env.HOPR_BOOTSTRAP_SERVERS !== undefined) {
+    addresses = process.env.HOPR_BOOTSTRAP_SERVERS.split(',')
+  }  else {
+    // Fall back to DNS
+    let records = await dns.promises.resolveTxt('_dnsaddr.bootstrap.develop.hoprnet.org')
```

Comment:
So ideally we have a way to read `ENVIRONMENT` and if not default to `develop`. Right now we have `_dnsaddr.bootstrap.develop` but we also have `_dnsaddr.bootstrap.testnet`, so we need a way to signal when we want which.

Reply:
what would you default to?  _dnsaddr.bootstrap.testnet? We can also add an ENV check in addition to manually adding bootstrap servers in the ENV var.